app/src/main/java/net/gsantner/opoc/format/GsTextUtils.java

```diff
@@ -163,17 +164,6 @@ public static int tryParseInt(final String value, int defaultValue) {
         }
     }
 
-    public static <T> ArrayList<T> toArrayList(T... array) {
-        ArrayList<T> list = new ArrayList<>();
-        Collections.addAll(list, array);
-        return list;
-    }
-
-    // Not null, not empty, not spaces only
-    public static boolean ne(final String str) {
-        return str != null && !str.trim().isEmpty();
```

Comment:
Redundant with `TextViewUtils.isNullOrWhitespace`

Reply:
some issues may come out of simple exchange of it. ne is the opposite.

Reply:
some issues may come out of simple exchange of it. ne is the opposite.

Like this change here: if (badMimeType && file.exists)

![Screenshot_20220928-202916](https://user-images.githubusercontent.com/6735650/192860875-410520f5-9bfb-4a8a-9bc1-7fb322978621.png)